src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
```

Comment:
Here the classes need to be adjust according to changes in html as suggested in previous comments.